### PR TITLE
[3.10] Show the target version to the pre upgrade checker

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #31292

### Summary of Changes

Show the target version to the pre upgeade checker

### Testing Instructions

- install 3.10
- set the custom update server to: `https://update.joomla.org/core/test/310to4_list.xml`
- check the pre upgrade checker

### Actual result BEFORE applying this Pull Request

image

### Expected result AFTER applying this Pull Request

image

### Documentation Changes Required

none

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 127958cffc54e47d6e53969f7b7b14be283c5f87<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32098">issues.joomla.org/tracker/joomla-cms/32098</a>.</sub>

File: administrator/language/en-GB/en-GB.com_joomlaupdate.ini [hunk 1]
```diff
@@ -99,7 +99,7 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_LIVE_UPDATE="Live Update is not available"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_LIVE_UPDATE_DESC="There is a new version of the Joomla Update Component that needs to be installed first. <a class=\"alert-link\" href="_QQ_"index.php?option=com_installer&view=update"_QQ_">Click here to update the component</a>."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_PACKAGE="Update package URL"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_PACKAGE_REINSTALL="Reinstall package URL"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_PREUPDATE_CHECK="Pre-Update Check"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_PREUPDATE_CHECK="Pre-Update Check for %s"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_joomlaupdate.ini]:
```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_PREUPDATE_CHECK="Pre-Update Check for Joomla %s"
```

It just reads better

Comment (zero-24) [reply administrator/language/en-GB/en-GB.com_joomlaupdate.ini]: Thanks

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on a41ed40eb9ec6c4918c6e48fdac1386ea84d7656<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32098">issues.joomla.org/tracker/joomla-cms/32098</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on a41ed40eb9ec6c4918c6e48fdac1386ea84d7656<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32098">issues.joomla.org/tracker/joomla-cms/32098</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32098">issues.joomla.org/tracker/joomla-cms/32098</a>.</sub>

**Comment by richard67** [comment]
Thanks!

**Pull request merged**